### ISSUE TITLE
Training Hyperparameter to Reproduce Paper Result

### ISSUE THREAD

**Issue by andrewivan123** [zhyever/PatchFusion#21]
I would like to reproduce the paper results so what are the training hyperparameters used in the paper (e.g batch size, no. of epochs, loss parameters, etc)?

**Comment by zhyever** [comment]
All the hyperparameters are given in the corresponding config files
